src/mobile/apps/auction_support/templates/registration-form.jade

```diff
@@ -11,8 +11,12 @@
     .registration-form-content
       include ../../../components/credit_card/templates/address
 
-  .registration-form-section.register
-    p
-      | By bidding, you agree to the&nbsp;
-      a( href="/conditions-of-sale" ) Conditions of Sale
-      | &nbsp;and that all information you provide is true and accurate.
+  .registration-form-section.accept-cos
```

Comment:
Can we extract just this view into a shared template? It looks like we'll need to use it for the `bid_form` cases as well so might be nice to abstract it out.

Reply:
Sure- do components shared between desktop/mobile go in desktop?

Reply:
Looks like there is a shared `components` folder, but I think `desktop` is fine for this kind of thing (since the responsive view would live there per our conventions)